userspace/libsinsp/test/sinsp_utils.ut.cpp

```diff
@@ -214,9 +215,12 @@ TEST(sinsp_utils_test, sinsp_split_check_terminator)
 	// check that the null terminator is enforced
 	const char *in = "hello\0worlddd";
 	size_t len = 13;
+#ifdef _DEBUG
+	EXPECT_THROW(sinsp_split(in, len, '\0'), sinsp_exception);
```

Comment:
In DEBUG mode this triggers a (well-known and desired) exception.
This fix allows to run tests on debug builds.